src/components/Header/index.tsx

```diff
@@ -22,7 +22,10 @@ const { Title } = Typography;
 const Header: React.FC<HeaderProps> = ({ user = null, location = "" }) => {
   const { t } = useTranslation(["common", "pages"]);
   const responsive = useResponsive();
+
   const isMobile = responsive.md === false;
+  const isSignUpPage = location === "sign_up";
+  const withLogo = WITHOUT_TITLE_ROUTES.includes(location);
```

Comment:
header should be dumb, but you're adding some business logic inside – it's incorrect. Let's provide `withLogo` as props